widget/button.go

```diff
@@ -118,10 +118,16 @@ func (b *buttonRenderer) BackgroundColor() color.Color {
 	switch {
 	case b.button.Disabled():
 		return theme.DisabledButtonColor()
-	case b.button.Style == PrimaryButton:
-		return theme.PrimaryColor()
+	case b.button.pressed:
```

Comment:
I guess that primary and default buttons should have a different pressed colour?